contract-derive/src/helpers.rs

```diff
@@ -128,3 +133,28 @@ where
         }
     }
 }
+
+// Helper function to generate the deployment code
+pub fn generate_deployment_code(
+    _struct_name: &Ident,
+    constructor: Option<&ImplItemMethod>,
+) -> quote::__private::TokenStream {
+    quote! {
+        use alloc::vec::Vec;
+
+        #[no_mangle]
+        pub extern "C" fn main() -> ! {
+            // TODO: figure out constructor
+
+            let runtime: &[u8] = include_bytes!("../target/riscv64imac-unknown-none-elf/release/runtime");
```

Comment:
I think this is fine for now, but we should at some point error out properly if this doesn't exist

Reply:
imo the macro's default panic message i already explicit enough:
```
error: couldn't read src/../target/riscv64imac-unknown-none-elf/release/inexistent_path: No such file or directory (os error 2)
  --> src/lib.rs:42:1
   |
42 | #[contract]
   | ^^^^^^^^^^^
   |
   = note: this error originates in the macro `include_bytes` (in Nightly builds, run with -Z macro-backtrace for more info)
```

thoughts?

Reply:
ah yes this looks good